main.gms

```diff
@@ -1474,6 +1474,8 @@ $setglobal cm_inco0RegiFactor  off  !! def = off
 *** cm_CCS_markup "multiplicative factor for CSS cost markup"
 ***   def <- "off" = use default CCS pm_inco0_t values.
 ***   or number (ex. 0.66), multiply by 0.66 the CSS cost markup
+$setglobal cm_ccsinjeCost low !! def = low !! regexp = med|low|high
```

Comment:
why are you using a compiler flag instead of a parameter?

Reply:
The idea was to have a name as we wanted to give a low, medium & high estimate.
Also, for the low estimate, which is the original one, it is necessary to change more than just inco0. But I guess I could also accomodate that with a parameter, and just suggest a value for the high estimate in main.gms, if you prefer that

Reply:
ah, o.k. I am fine with a compiler flag if there are reasons for that